### DISCUSSION TITLE
Discussions: A place to talk mpmd_marlin_1.1.x

### DISCUSSION THREAD

**Discussion by aegean-odyssey** [aegean-odyssey/mpmd_marlin_1.1.x#44] [General]
## Welcome!

> ### Please feel free to participate in any of the threads in the Discussions area or use the `New Discussion` button to begin a new discussion and topic.

Welcome to a place to discuss anything related to `mpmd_marlin_1.1.x` project. Ask questions, share ideas, relay some news, ... all those communications that aren't quite _reporting a firmware issue_ and not quite _documentation for the wiki_.

December 2020. The project is a little more than a year old now, and with the firmware mostly working, perhaps it's time to concentrate on some of the ancillary projects for the firmware and the printer:

*  tools to assist the auto-calibration, [mpmd-calibration-experimental](/aegean-odyssey/mpmd-calibration-experimental);
* a PrusaSlicer vendor configuration bundle, [PrusaSlicer-settings](/aegean-odyssey/PrusaSlicer-settings); and
* some add-ons I find useful, such as [mpmd-bed-chock](/aegean-odyssey/mpmd-bed-chock) and [mpmd-tower-shield](/aegean-odyssey/mpmd-tower-shield);

The setup here is deliberately simple, keeping with the stock printer components along with the `mpmd_marlin_1.1.x` firmware and the add-ons mentioned above. I also use a glass plate on the build surface -- nothing fancy, attached with double-sided tape. I usually print on masking tape and calibrate and configure the printer with only commands from the `setup_gcode` folder of the microSD card contents. Models are sliced with PrusaSlicer as it's easier to use than Cura on my low-end linux laptop. Aside from the atrocious first layer, the prints are good and using the printer is just plain fun for the most part. The goal is to squeeze as much performance as possible out of the mainly stock MP Mini Delta printer.

Anyway, the forum is open. Let's see if we can do something useful with this new "Discussions" feature of GitHub.

And at the risk of being terribly dated:
>"Don't be mean; we don't have to be mean, cause, remember, no matter where you go, there you are." -- Buckaroo Bonzai, The Adventures of Buckaroo Banzai Across the 8th Dimension (1984) .

**Comment by drji3m** [comment]
I  installed on mynew delta.. Great work on firmware.

But im ready to return the delta since the usb serial is totally worthless.
Most times octoprint wont connect, other  times it fails while printing.

it gets worse. If i reboot the pi the printer hangs the pi and it cant reboot. Monoprice lives in a world of its own and cant make a simple USB interface work. The mainboard must go.

But myquestion to you is whether you think your firmware will install on a creality ender 3 maniboard-its a stm 32 processor.  Im willing to try this swap before  i give up on Monoprice.

**Reply by drji3m** [reply]
ender 3 pro stm32f103ret6

**Reply by aegean-odyssey** [reply]
[ locked this discussion, as it is more or less an annoucement ] 
I started a new discussion with the comments, please see Octoprint #50
